plottable.d.ts

```diff
@@ -1502,6 +1503,24 @@ declare module Plottable {
          */
         anchor(selection: D3.Selection): Component;
         /**
+         * Adds a callback to be called on anchoring the Component to the DOM.
+         * If the component is already anchored, the callback is called immediately.
+         *
+         * @param {AnchorCallback} callback The callback to be called on Anchor
+         *
+         * @return {Component}
+         */
+        onAnchor(callback: AnchorCallback): Component;
+        /**
+         * Removes a callback to be called on anchoring the Component to the DOM.
+         * The callback is identified by reference equality.
+         *
+         * @param {AnchorCallback} callback The callback to be removed
```

Comment:
I'm trying to find a better parallel between the `onAnchor` and `offAnchor` docs and it mostly looks good.

For this param part, would it be too vague to say
The callback to be added / removed ?

I guess another way to word it is that the parallel doc for `offAnchor` are a little odd, `The callback to be called off Anchor`